Lesson3/src/Weather.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable indent */
+//import React from 'react';
+
+export default class Weather {
+    constructor() {
+        this.fetchURL = 'http://api.openweathermap.org/data/2.5/weather?q=Moscow,ru&lang=ru&units=metric&APPID=664f14ad10ba07afd39b7103b1a1c196';
+    }
+
+    getCurrentWeather() { // Не выходит получить данные отсюда в модуле Login, получаю undefined(т.к. функция не успевает вернуть результат) либо ошибку с then т.к. похоже в конце не промис же выдаю, но пока я только приближаюсь к пониманию "что почем" здесь
+        fetch(this.fetchURL)
```

Comment:
вам просто оставалось вернуть резултат fetch'a (т.е. promise) 😄 
перед fetch добавьте слово return
```suggestion
        return fetch(this.fetchURL)
```